tests/data/blazemark-output.txt

```diff
@@ -0,0 +1,48 @@
+ Dense Vector/Dense Vector Addition:
```

Comment:
Did you forget to git add a test that uses this? I don't see this getting used anywhere